sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.1.0-beta.8 (Unreleased)
 
 ### Breaking Changes
+- Removed `StringIndexType`, the type will be shared in the Azure Core.
```

Comment:
Very late to the party, but it will be better to not make promises in the Cnahgelog as we don't know what could actually happen. Consider just saying that the type has been removed and the default encoding for the library is ...

Reply:
address it in https://github.com/Azure/azure-sdk-for-java/pull/22329/files#diff-7fbca8ee7335dc54eff4896d71e776956633f83aa352534da69dca097fefe74fR12